src/tqdm_publisher/publisher.py

```diff
@@ -9,22 +9,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.callbacks = {}
 
+
     # Override the update method to call callbacks 
-    def update(self, n=1, always_callback=False):
-        if super().update(n) or always_callback:
+    def update(self, n=1):
```

Comment:
```suggestion
    def update(self, n: int=1) -> Union[bool, None]:
```

Might need to add `from typing import Union` at top

Reply:
Good catch. Thanks!